### PR TITLE
systemtests fixes

### PR DESCRIPTION
**Pull request by unclejack**
This PR addresses some issues with systemtests. The systemtests were failing for no obvious reason. No errors were being logged anywhere. The second commit addresses this problem and it makes the systemtests throw errors when the cfg.json is invalid.

The third commit changes the committed cfg.json to serve as an example and alters the config generator to generate functional config files.

### CODE REVIEW

File: test/systemtests/util_test.go [hunk 1]
```diff
@@ -1066,19 +1066,30 @@ func (s *systemtestSuite) verifyIPs(ipaddrs []string) error {
...
+		return b, c, d, err
+	}
 	json.Unmarshal(raw, &c)
-	var d GlobInfo
+	if err != nil {
+		return b, c, d, err
+	}
 	json.Unmarshal(raw, &d)
-	return b, c, d
+	if err != nil {
```
Comment (neelimamukiri) [inline test/systemtests/util_test.go]: Isn't this if redundant? Why not just return b, c, d, err?

**Comment by neelimamukiri** [comment]
Please also update https://github.com/contiv/netplugin/blob/master/test/systemtests/How-to-Run.md with the key_file to keyFile configuration.

**Comment by unclejack** [comment]
@neelimamukiri It's done. PTAL

File: test/systemtests/init_test.go [hunk 2]
```diff
@@ -80,8 +84,12 @@ func TestSystem(t *T) {
 }
 
 func (s *systemtestSuite) SetUpSuite(c *C) {
+	var err error
 	logrus.Infof("Bootstrapping system tests")
-	s.basicInfo, s.hostInfo, s.globInfo = getInfo("cfg.json")
+	s.basicInfo, s.hostInfo, s.globInfo, err = getInfo("cfg.json")
+	if err != nil {
+		logrus.Infof("failed to load cfg.json: %v", err)
```
Comment (neelimamukiri) [inline test/systemtests/init_test.go]: Do we need an exit here as well?

Comment (unclejack) [inline test/systemtests/init_test.go]: It's done. PTAL when you have the time.

**Comment by unclejack** [comment]
The CI failures have nothing to do with the code.

**Comment by unclejack** [comment]
@jojimt Please take a look to review this and merge when you have the time.

**Pull request merged**